gpu/octree/src/cuda/knn_search.cu

```diff
@@ -90,13 +91,13 @@ namespace pcl { namespace device { namespace knn_search
         int query_index;        
         float3 query;  
         
-        float min_distance;
+        float min_distance, min_distance2;
```

Comment:
Not to be that guy, but is this `min_distance_squared`, `min_distance_tmp`, `second_smallest_distance` or something else?

Reply:
Good point, it should be `min_distance_squared`, `mindistance2` is pretty amibguous. Will change it.